package.json

```diff
@@ -7,6 +7,7 @@
         "test": "jest",
         "build": "tsc",
         "lint": "eslint --ext .js,.ts,.tsx .",
+        "start:dev": "dotenv -e ../.env -- nodemon --watch 'src/**/*.ts' --ignore 'src/**/*.spec.ts' --exec 'ts-node' src/main.ts | pino-pretty",
```

Comment:
There seems to be some convention that this is just called `dev` ... just sayin'